File: frontend/lib/Models/PersonnelProperty/PersonnelPropertyListResponse.dart
class PersonnelPropertyListResponse {
  final int propertyID;
  final String dueOn;
  final bool isWaiting;

  const PersonnelPropertyListResponse(
      {required this.propertyID, required this.dueOn, required this.isWaiting});

  factory PersonnelPropertyListResponse.fromJson(
      Map<String, dynamic> jsonData) {
    return PersonnelPropertyListResponse(
        propertyID: jsonData["propertyID"],
        dueOn: jsonData["dueOn"],
        isWaiting: jsonData["isWaiting"]);
  }
}

File: backend/InventoryDemo.DTOs/Responses/PersonnelPropertyListResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InventoryDemo.DTOs.Responses
{
    public class PersonnelPropertyListResponse
    {
        public int PropertyID { get; set; }
        public DateTime DueOn { get; set; }
        public bool IsWaiting { get; set; }
    }
}
